### ISSUE TITLE
Feature request: structuring control panel

### ISSUE THREAD

**Issue by aloboa** [r-spatial/leafem#91]
Being able to structure busy control panels "à la" QGIS, with groups and subgroups, would be very good. 
Optimally, both groups and subgroups could be defined as having display/not display buttons, but a least
just being able to indent groups under generic (best foldable) titles would result in more tidy displays.

**Comment by trafficonese** [comment]
I am not entirely sure what you are trying to do, as in `leaflet::addControl` you can create custom HTML, so you should be able to do everything.

If you're looking for a Grouped LayerControl, then you could check out the latest version of [leaflet.extras](https://github.com/trafficonese/leaflet.extras) which now supports [leaflet-groupedlayercontrol](https://github.com/trafficonese/leaflet-groupedlayercontrol).

See [this example](https://github.com/trafficonese/leaflet.extras/blob/master/inst/examples/shiny/groupedlayercontrol_app.R)

**Comment by aloboa** [comment]
The examples such as https://ish.lol/leaflet-groupedlayercontrol/example/basic.html do exactly what I suggest,
but what I am requesting is doing it through an R function, not having to create custom HTML.

**Comment by trafficonese** [comment]
Yes, [`leaflet.extras`](https://github.com/trafficonese/leaflet.extras) is an R package.

Check out this Shiny example:
https://github.com/trafficonese/leaflet.extras/blob/master/inst/examples/shiny/groupedlayercontrol_app.R

or here:
https://trafficonese.github.io/leaflet.extras/reference/addGroupedLayersControl.html

**Comment by aloboa** [comment]
Thanks,
but https://github.com/trafficonese/leaflet.extras/issues/228
